config/routes.rb

```diff
@@ -1,6 +1,17 @@
 Rails.application.routes.draw do
   root 'home#index'
   resources :cards
-  resources :users
+  # resources :users
+  resources :user_sessions
+  resources :registrations, only: [:new, :create]
+  resources :profiles, only: [:index, :show, :edit, :update, :destroy]
+  get 'login' => 'user_sessions#new', as: :login
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.